code/modules/mob/living/carbon/human/examine.dm

```diff
@@ -343,6 +343,19 @@
 	else if(isobserver(user))
 		. += span_info("<b>Traits:</b> [get_quirk_string(FALSE, CAT_QUIRK_ALL)]")
 
+	var/flavor_text_link
+	/// The first 1-FLAVOR_PREVIEW_LIMIT characters in the mob's "flavor_text" DNA feature. FLAVOR_PREVIEW_LIMIT is defined in flavor_defines.dm.
+	var/preview_text = copytext_char((dna.features["flavor_text"]), 1, FLAVOR_PREVIEW_LIMIT)
```

Comment:
Really not a fan of this being stored in DNA.features.

Reply:
As long as storing it elsewhere does not replicate the problem I have seen in other servers where when the client disconnects, their flavor text doesnt go with the client which causes weird issues. @Kapu1178

Reply:
May be best to store it in mind instead.